README.md

```diff
@@ -160,6 +160,32 @@ cargo read <day>
 # ...the input...
 ```
 
+### Scaffold, download and read in one go
+
+This command runs `cargo scaffold`, `cargo download` and `cargo read` for the current day. You can also specify the specific day yourself in an optional argument.
+
+```sh
+# example: `cargo today` on December 1st
+cargo today [<day>]
```

Comment:
We added the `--download` flag to `solve` in the mean time, so we should remove it here. I think that having a clear separation makes the most sense - have `solve` work anytime and `today` be automatic in december.